streams/src/test/java/org/apache/kafka/streams/processor/internals/RecordCollectorTest.java

```diff
@@ -199,6 +200,44 @@ public void shouldThrowStreamsExceptionOnAnyExceptionButProducerFencedException(
         collector.send("topic1", "3", "0", null, null, stringSerializer, stringSerializer, streamPartitioner);
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
What is the reason for this?

Reply:
Missing generics in the method. I fixed it.